CHANGES.rst

```diff
@@ -222,6 +222,11 @@ Bug Fixes
 
 - ``astropy.io.fits``
 
+  - Handle unicode FITS BinTable column names on Python 2 [#5204, #4805]
```

Comment:
@weaverba137 Did you write that changelog entry or is it an artificat from the rebase?